src/prommis/solvent_extraction/ree_aq_distribution.py

```diff
@@ -158,5 +158,17 @@ def get_material_flow_terms(self, p, j):
                 to_units=units.mol / units.hour,
             )
 
+    def get_material_density_terms(self, p, j):
+        if j == "H2SO4":
```

Comment:
This will get fixed by @ArkoprabhoDasgupta, but note that this makes some very big assumptions that are likely wrong. This will only affect dynamic simulations however.